src/axom/primal/operators/compute_bounding_box.hpp

```diff
@@ -189,6 +190,23 @@ AXOM_HOST_DEVICE BoundingBox<T, NDIMS> compute_bounding_box(
   return BoundingBox<T, NDIMS> {tet[0], tet[1], tet[2], tet[3]};
 }
 
+/*!
+ * \brief Creates a bounding box around a Polygon
+ *
+ * \param [in] poly The polygon
+ */
+template <typename T, int NDIMS>
+AXOM_HOST_DEVICE BoundingBox<T, NDIMS> compute_bounding_box(
```

Comment:
Too bad the BoundingBox needs the T, NDIMS template arguments or a single definition of compute_bounding_box could suffice. These seems like the easiest way considering Polyhedron and Polygon do not share inheritance.